src/app/modules/shared_models/message_item.nim

```diff
@@ -135,13 +138,16 @@ proc initItem*(
   result.quotedMessageParsedText = quotedMessageParsedText
   result.quotedMessageContentType = quotedMessageContentType
   result.quotedMessageDeleted = quotedMessageDeleted
-  result.quotedMessageFromIterator = 0
+  result.quotedMessageAuthorDetails = quotedMessageAuthorDetails
 
   if quotedMessageContentType == ContentType.DiscordMessage.int:
     result.quotedMessageAuthorDisplayName = quotedMessageDiscordMessage.author.name
     result.quotedMessageAuthorAvatar = quotedMessageDiscordMessage.author.localUrl
     if result.quotedMessageAuthorAvatar == "":
       result.quotedMessageAuthorAvatar = quotedMessageDiscordMessage.author.avatarUrl
+  else:
```

Comment:
@0x-r4bbit I've just resolved the conflicts with #9111. What do you think, does it makes sense to do this, or it's redundant?

Reply:
It doesn't really make sense because:

1. If there's no localUrl for the author in question, there will be an avatarUrl (so this will be set regardless)
2. There will be no proper displayName and thumbnail in authorDetails for this type of message because we don't have contact data from discord messages

Reply:
What I've done here is to also fill result.quotedMessageAuthorDisplayName and result.quotedMessageAuthorAvatar for all other quoted messages. There should be no change in the discord quoted message display name and avatar. But I was worried that the author data might be duplicated in quotedMessageDiscordMessage.author and quotedMessageAuthorDetails.

Reply:
Ah, sorry misread the indentation.
So, I do know we hydrate `quotedMessageAuthorDetails` later in QML via contact data. If we can be sure that at this point in time the data is available, I think the change is fine.

You can verify this easily by replying to messages and checking if the reply-to-message has all the author information as needed.